From f8064ed09d09f1a502d18e77f1b1bbe383a99251 Mon Sep 17 00:00:00 2001
From: Santiago Rossi <santiago_rho_psi@hotmail.com>
Date: Mon, 16 Apr 2012 21:20:46 -0300
Subject: [PATCH 22/40] 4.4.3 Cuenca de Sholazar: El rayo golpea dos veces sin
 duda

Fix for quest "Lightning Definitely Strikes Twice" from Oracles/Frenzyheart quest chain
---
 ...104_world_lightning_definetly_strikes_twice.sql |   11 +++++++++++
 1 files changed, 11 insertions(+), 0 deletions(-)
 create mode 100644 sql/updates/world/reanemu/2012_04_16_104_world_lightning_definetly_strikes_twice.sql

diff --git a/sql/updates/world/reanemu/2012_04_16_104_world_lightning_definetly_strikes_twice.sql b/sql/updates/world/reanemu/2012_04_16_104_world_lightning_definetly_strikes_twice.sql
new file mode 100644
index 0000000..25fd21c
--- /dev/null
+++ b/sql/updates/world/reanemu/2012_04_16_104_world_lightning_definetly_strikes_twice.sql
@@ -0,0 +1,11 @@
+-- Lightning Definitely Strikes Twice
+DELETE FROM `gameobject` WHERE `id`=190502;
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`) VALUES
+(NULL,190502,571,1,1,6206.71,4951.67,-98.9126,3.84683,0,0,0.938472,-0.345355,-60,0,1);
+
+DELETE FROM `event_scripts` WHERE `id` IN (18474,18475);
+INSERT INTO `event_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `dataint`, `x`, `y`, `z`, `o`) VALUES 
+(18474,2,7,12537,0,0,0,0,0,0),
+(18474,1,10,28253,6000,0,6207.35937,4952.259277,-81.744415,0.748111);
+INSERT INTO `event_scripts` (`id`,`delay`,`command`,`datalong`,`datalong2`)
+SELECT 18475,1,9, `guid`,60 FROM `gameobject` WHERE `id`=190502;
\ No newline at end of file
-- 
1.7.9.msysgit.0

